cluster/gce/trusty/node.yaml

```diff
@@ -162,8 +163,16 @@ start on stopped kube-env
 
 script
 	. /etc/kube-env
-	# If kubelet or kube-proxy is not installed in the image, pull release binaries and put them in /usr/bin.
-	if ! which kubelet > /dev/null || ! which kube-proxy > /dev/null; then
+	# If both kubelet and kube-proxy test args are set, this is a testing cluster.
+	# We pull binaries and put them in /usr/local/bin. If it is not a testing
```

Comment:
Some GCE images do not have a writable /usr/bin. So, /usr/local/bin is an alternative place.